### DISCUSSION TITLE
键盘上下键音量可自己调整

### DISCUSSION THREAD

**Discussion by HornyPrivateGamer** [the1812/Bilibili-Evolved#5043] [功能建议 / Ideas]
目前如果按上下键的话是增加10%或减少10%

增加百分之十对于某些人来说这太多了像YouTube的话是百分之五

所以希望能增加一个功能, 能自由调整自己想增加的音量大小

目前已捐三十块, 如果解决问题的话还能给50

谢谢大佬 @the1812 

![Screenshot 2024-11-21 at 4 42 41 PM 1](https://github.com/user-attachments/assets/594c58d7-3dda-46fe-8d79-119173881097)

**Comment by the1812** [comment]
![Snipaste_2024-11-21_22-48-01](https://github.com/user-attachments/assets/12613e73-d03a-4c11-a4e9-36f6bc9fc7e9)
键盘用快捷键扩展就支持的
